src/components/popups/TeamSetting.tsx

```diff
@@ -206,6 +148,13 @@ const TeamSettings = ({ isOpen, onClose, onEdit, setIsLeave }) => {
     }
   };
 
+  const getAllErrorMessages = () => {
+    const fieldErrors = Object.values(errors).filter((error) => error);
+    if (generalError) fieldErrors.push(generalError);
+
+    return fieldErrors;
+  };
+
```

Comment:
seems like a general function that can be used in other places as well. maybe appropriate to add to `FormField` component?

Reply:
Done